src/topp/IDMapper.cpp

```diff
@@ -262,12 +267,9 @@ class TOPPIDMapper :
         MzMLFile().load(spectra, exp);
       }
 
-      mapper.annotate(map, peptide_ids, protein_ids,
-                      (getStringOption_("feature:use_centroid_rt") == "true"),
-                      (getStringOption_("feature:use_centroid_mz") == "true"),
-                      exp);
+      mapper.annotate(map, peptide_ids, protein_ids, (getStringOption_("feature:use_centroid_rt") == "true"), (getStringOption_("feature:use_centroid_mz") == "true"), exp);
```

Comment:
found the formatting on the left better to read... but well not so important. I personally don't use reformat because of these issues. hope this get's better at some point

Reply:
extra cookie points if someone looks into https://clang.llvm.org/docs/ClangFormatStyleOptions.html to check options for keeping the alignment of long function parameter lists. ;)

Reply:
but looks like only: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#alignafteropenbracket exist and that one also looked strange sometimes